pkg/jailer/jailer.go

```diff
@@ -77,20 +77,9 @@ func CreateJail(name string) error {
 	return nil
 }
 
-func WhitelistEnvvars(envvars []string) []string {
-	wl := settings.WhitelistEnvironmentVars.Get()
-	envWhiteList := strings.Split(wl, ",")
-
-	if len(envWhiteList) == 0 {
-		return envvars
-	}
-
-	for _, wlVar := range envWhiteList {
-		wlVar = strings.TrimSpace(wlVar)
-		if val := os.Getenv(wlVar); val != "" {
-			envvars = append(envvars, wlVar+"="+val)
-		}
-	}
-
+func getWhitelistedEnvVars(envvars []string) []string {
```

Comment:
Nice that you made it unexported, no reason for it to be exported in `jailer` pkg.